test/compilable/test17906.d

```diff
@@ -3,5 +3,5 @@
 deprecated void main ()
 {
     Object o = new Object;
-    delete o;
+    destroy(o);
 }
```

Comment:
This test is explicitly checking that `delete` can be used in a deprecated scope, so there's no need to keep this file when removing `delete`.

Reply:
Noted, I've removed it in the follow-up to this.

Reply:
(https://github.com/dlang/dmd/commit/60a401a8dea8ca56afa66f6f208d56b58ecce88e)